docs/content/app-protect-dos/configuration.md

```diff
@@ -8,7 +8,7 @@ toc: true
 ---
 
 This document describes how to configure the NGINX App Protect Dos module
-> Check out the complete [NGINX Ingress Controller with App Protect Dos example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.1.1/examples/appprotect-dos).
+> Check out the complete [NGINX Ingress Controller with App Protect Dos example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.1.2/examples/appprotect-dos).
```

Comment:
```suggestion
> Check out the complete [NGINX Ingress Controller with App Protect DoS example resources on GitHub](https://github.com/nginxinc/kubernetes-ingress/tree/v2.1.2/examples/appprotect-dos).
```

Reply:
Looks like this needs to be fixed throughout the doc, this can be handled as a separate bug if desired.

Reply:
I've updated in all the docs